disco_aws_automation/disco_aws.py

```diff
@@ -592,7 +607,10 @@ def spindown(self, hostclasses):
         .. warning:: This currently does a dirty shutdown, no attempt is made to preserve logs.
         """
         for hostclass in hostclasses:
-            self.autoscale.delete_groups(hostclass=hostclass, force=True)
+            if is_truthy(self.config('spotinst', hostclass)):
+                self.elastigroup.delete_groups(hostclass)
```

Comment:
This should try to delete the group in both places regardless of the config. The config might have been changed after the group was spun up